Cargo.toml

```diff
@@ -23,6 +23,9 @@ maintenance = { status = "actively-developed" }
 name = "cargo-deny"
 path = "src/cargo-deny/main.rs"
 
+[features]
+vendored-openssl = ["rustsec/vendored-openssl"]
```

Comment:
Can you make this a default feature, that way musl builds will still work correctly (the reason for this feature in the first place). Then if you want to use the system openssl you can just pass `--no-default-features` when building/installing it to get the behavior you want.

Reply:
Sure, I'll do that.